website/docs/cloud-docs/api-docs/state-versions.mdx

```diff
@@ -46,6 +46,7 @@ Some of the information returned in a state version API object might be **popula
 
 Attribute                        | Description
 ---------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+`billable-rum-count`             | Count of billable RUM from resources. Only present for Org members on RUM plans who have visibility to see billable rum usage in the Usage page
```

Comment:
Do you think we should say `resources under management` instead of RUM so it's clearer? We also may want to say organization instead of org

Reply:
Yeah, that or we can say `Resources Under Measurement (RUM)` and then use `RUM` later in the explanation. But agreed that it's likely better in the public docs to clarify these things. We should capitalize the acronym when we use it later on too.